spec.emu

```diff
@@ -64,17 +64,56 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
                 1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; a newly created *TypeError* object &raquo;).
                 1. Return _promiseCapability_.[[Promise]].
               1. Set _phase_ to ~source~.
-          1. Let _moduleRequest_ be a new ModuleRequest Record { [[Specifier]]: _specifierString_, [[Phase]]: _phase_ }.
-          1. Perform HostLoadImportedModule(_referrer_, _moduleRequest_, ~empty~, _promiseCapability_).
+          1. Let _state_ be the DynamicImportState Record { [[PromiseCapability]]: _promiseCapability_, [[Phase]]: _phase_ }.
+          1. Perform HostLoadImportedModule(_referrer_, _specifierString_, ~empty~, _state_).
           1. Return _promiseCapability_.[[Promise]].
         </emu-alg>
       </emu-clause>
 
+      <p><ins>A <dfn id="dynamicimportstate-record" variants="DynamicImportState Records">DynamicImportState Record</dfn> is a Record that contains information about the loading process of a module graph. It's used to continue loading after a call to HostLoadImportedModule. Each DynamicImportState Record has the fields defined in <emu-xref href="#table-dynamicimportstate-record-fields"></emu-xref>:</ins></p>
+      <emu-table id="table-dynamicimportstate-record-fields" caption="DynamicImportState Record Fields">
```

Comment:
Another issue I have with this approach is adding more difference between dynamic import and normal import paths. As much as possible it would be great to move towards unification of these paths not divergence.